openvdb_points/tools/AttributeArray.h

```diff
@@ -223,7 +225,7 @@ class AttributeArray
 
 public:
     enum Flag { TRANSIENT = 0x1, HIDDEN = 0x2, GROUP=0x4, WRITEUNIFORM=0x8,
-                WRITEMEMCOMPRESS=0x10, WRITEDISKCOMPRESS=0x20, OUTOFCORE=0x40 };
+                WRITEMEMCOMPRESS=0x10, WRITEDISKCOMPRESS=0x20, OUTOFCORE=0x40, STRING=0x80 };
```

Comment:
I assume string attribute arrays suffer from the same issue as group attribute arrays when constructing a "compatible" attributeset from an existing descriptor (e.g. the flags aren't stored on the descriptor, and so the group and string arrays do not have their flags set). I feel this problem gets worse as we add more subclasses that are tracked using these flags - could we find a way of addressing this issue? (both for strings and groups). Perhaps it would even be worth removing the ability to construct a new attributeset from a descriptor, given the pitfalls...